CHANGELOG.md

```diff
@@ -2,7 +2,12 @@
 
 This file is used to list changes made in each version of the Java cookbook.
 
-## Unreleased
```

Comment:
Apprehensive about removing the `Unreleased` tag, because it's technically not released in the upstream and this was method by way of denoting that (and I think was a convention a year ago).
Might be worth keeping it or denoting `Roblox Release` or something

Reply:
Oh, didn't realize that
Fixing

Reply:
@Brantone Updated